README.md

```diff
@@ -3,10 +3,28 @@
 [![Travis CI Status](https://secure.travis-ci.org/openlayers/ol3.svg)](http://travis-ci.org/#!/openlayers/ol3)
 [![Coverage Status](https://coveralls.io/repos/openlayers/ol3/badge.svg?branch=master)](https://coveralls.io/r/openlayers/ol3?branch=master)
 
-Welcome to [OpenLayers 3](http://openlayers.org/)!
+[OpenLayers 3](http://openlayers.org/) is a high-performance, feature-packed library for creating interactive maps on the web.
 
-## Examples
-Check out the [hosted examples](http://openlayers.org/en/master/examples/), the [workshop](http://openlayers.org/ol3-workshop/) or poke around the evolving [API docs](http://openlayers.org/en/master/apidoc/).
+## Getting Started
+
+- Download the [latest release](https://github.com/openlayers/ol3/releases/tag/v3.4.0)
```

Comment:
We need to edit https://github.com/openlayers/ol3/wiki/Release-Procedure and document that this link should be changed when creating a new release.